data/aliases.js

```diff
@@ -489,6 +489,7 @@ exports.BattleAliases = {
 	"amph": "Ampharos",
 	"azu": "Azumarill",
 	"bliss": "Blissey",
+	"birdjesus": "Pidgeot",
```

Comment:
No, the Twitch streamer used it in the Past Gens and that's where it was named but I can make it mega if wanted